cfme/control/explorer/actions.py

```diff
@@ -227,31 +227,30 @@ class ActionCollection(BaseCollection):
 
     def create(self, description, action_type, action_values=None):
         """Create an Action in the UI."""
-        action_values = action_values or {}
+        action = self.instantiate(description, action_type, action_values=action_values)
```

Comment:
**QUESTION**
@mfalesni  @psav should we instantiate an object before its creation in the UI? Does it matter?

Reply:
@quarckster in this case it matters, because ```alerts_to_evaluate``` are converted from list of Alerts to strings.
Without this it throws TypeError: unhashable type: 'Alert'.
I've looked at this for reference: [link](https://github.com/ManageIQ/integration_tests/blob/master/cfme/control/explorer/alert_profiles.py#L211)